### PR TITLE
fix: transfer ownership userstory completed [1118]

### PR DESCRIPTION
**Pull request by isha-dubey**
## Description

#1118 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts [hunk 1]
```diff
@@ -416,31 +416,39 @@ export class TeamExplorerPageViewModel {
...
+          `Failed to update access of Owner. Please try again.`,
+        );
+      }
+      return response;
     } else {
-      notifications.error(
-        `Failed to update access of Owner. Please try again.`,
-      );
+      notifications.error("You need more than one team to transfer ownership");
+      return;
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts:449]: @isha-dubey I think we don't need more than one team to transfer ownership, could you please elaborate the scenario.

File: src/packages/@teams/features/team-members/components/member/Member.svelte [hunk 2]
```diff
@@ -296,10 +303,20 @@
...
+      <div
+        class="d-flex align-items-center justify-content-center"
+        style="width: 36px;
+      border: 1px solid var(--border-color);
+      height: 36px;
+      border-radius: 50%;"
+      >
+        <span>{openTeam?.name[0].toUpperCase()}</span>
+      </div>
+    {/if}
```
Comment (itsmdasifraza) [inline src/packages/@teams/features/team-members/components/member/Member.svelte:319]: lets convert it to reusable component named `IconFallback` in "library/ui" using props - 
height, width, backgroundColor, borderColor, textColor, fontSize

use js docs, and hocs

File: src/packages/@teams/features/team-members/components/member/Member.svelte [hunk 3]
```diff
@@ -350,22 +367,36 @@
...
+      {:else}
+        <div
+          class="d-flex align-items-center justify-content-center"
+          style="width: 36px;
+        border: 1px solid var(--border-color);
+        height: 36px;
+        border-radius: 50%;"
+        >
+          <span>{openTeam?.name[0].toUpperCase()}</span>
+        </div>
```
Comment (itsmdasifraza) [inline src/packages/@teams/features/team-members/components/member/Member.svelte:395]: use "IconFallback" component here also

File: src/packages/@teams/features/team-members/components/member/Member.svelte [hunk 4]
```diff
@@ -444,12 +481,26 @@
...
+                alt=""/>
+      {:else}
+        <div
+          class="d-flex align-items-center justify-content-center"
+          style="width: 36px;
+        border: 1px solid var(--border-color);
+        height: 36px;
+        border-radius: 50%;"
+        >
+          <span>{openTeam?.name[0].toUpperCase()}</span>
```
Comment (itsmdasifraza) [inline src/packages/@teams/features/team-members/components/member/Member.svelte:502]: use "IconFallback" component here also

File: src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts [hunk 5]
```diff
@@ -416,31 +416,39 @@ export class TeamExplorerPageViewModel {
...
-    );
-    if (response.isSuccessful === true) {
-      const responseData = response.data.data;
-      await this.teamRepository.modifyTeam(_teamId, responseData);
-      await this.refreshWorkspaces(_userId);
-      notifications.success(
-        `${_userName} is now the new Owner of ${_teamName}.`,
+    const res = await this.teamRepository.getTeamData();
+    console.log("This is res", res.length);
+
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts:427]: console not required!

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Comment by itsmdasifraza** [comment]
please use issue hash in commit message ex-
`feat: commit message [#1095]`

File: src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts (hunk 1 above)
Comment (isha-dubey) [reply src/packages/@app/pages/TeamExplorerPage/TeamExplorerPage.ViewModel.ts:449]: Let's connect , it's a little complicated to explain in comments.

**Comment by isha-dubey** [comment]
Added the changes for user-story #1120  in this PR .

**Review by itsmdasifraza — APPROVED** [review]

**Review by gc-codes — APPROVED** [review]

**Pull request merged**